README.md

```diff
@@ -43,7 +45,7 @@ This provider's versions are able to install and manage the following versions o
 | Kubernetes 1.20 | ✓ | ✓ | ✓ |
 | Kubernetes 1.21 |  | ✓ | ✓ |
 | Kubernetes 1.22 |  |  | ✓ |
- 
+<p align="center"><img alt="capi" src="https://cloud.google.com/_static/cloud/images/favicons/onecloud/super_cloud.png" width="200x" /></p> 
```

Comment:
i did not get why this? this does not show up in any place, at least i could not see it